openupgrade_scripts/scripts/point_of_sale/14.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,56 @@
+from openupgradelib import openupgrade
+
+
+def convert_field_m2o_to_o2m(env):
+    openupgrade.m2o_to_x2m(
+        env.cr, env["pos.order"], "pos_order", "picking_ids", "picking_id"
+    )
+
+
+def fill_sesion_on_picking_by_pos_order(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE stock_picking
+        SET pos_session_id = (SELECT session_id
+                                FROM pos_order
+                                WHERE pos_order.id = stock_picking.pos_order_id
+                                LIMIT 1)
+        WHERE pos_session_id IS NULL AND pos_order_id IS NOT NULL;
```

Comment:
Simplify this expression:

```suggestion
        UPDATE stock_picking sp
        SET pos_session_id =  po.session_id
        FROM pos_order po
        WHERE po.id = sp.pos_order_id
            AND sp.pos_session_id IS NULL
```

Reply:
In fact, when I made the PR I didn't checked if the queries could be simplified. Thanks